recipes/leftbar/callbox/callbox.vue

```diff
@@ -199,26 +194,9 @@ export default {
   background-color: var(--dt-color-surface-primary);
   border-radius: var(--dt-size-radius-300);
 
-  &:deep(.dt-item-layout--left) {
-    justify-content: flex-start;
-    padding-right: 0;
-  }
-
-  &:deep(.dt-item-layout--content) {
-    overflow: hidden;
-  }
-
-  &:deep(.dt-item-layout--bottom) {
-    margin-top: 0;
-  }
-
-  &:deep(.dt-item-layout--right) {
-    flex-shrink: 0;
-  }
-
   &--video {
     border-radius: var(--dt-size-radius-200) var(--dt-size-radius-200) 0 0;
-    overflow: hidden;
+    overflow: clip;
```

Comment:
All this `<style>` comes from initially the `dt-list-item` component and you update it a little bit?

Reply:
No,  those custom styles are needed, with or without the `dt-item-layout`